src/Actions/Members/DeleteMemberAction.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Xguard\LaravelKanban\Actions\Members;
+
+use Illuminate\Contracts\Foundation\Application;
+use Illuminate\Contracts\Routing\ResponseFactory;
+use Illuminate\Http\Response;
+use Illuminate\Support\Facades\Auth;
+use Xguard\LaravelKanban\Models\Log;
+use Xguard\LaravelKanban\Models\Member;
+
+use Lorisleiva\Actions\Action;
+
+class DeleteMemberAction extends Action
+{
+
+    /**
+     * Execute the action and return a result.
+     *
+     * @return Application|ResponseFactory|Response
+     */
+    public function handle()
```

Comment:
you can add return type at the end of handle ie: public function handle() : Response